A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_VerticalRevisionPage.js

```diff
@@ -111,4 +119,11 @@ class CDUVerticalRevisionPage {
             };
         }
     }
+
+    static formatFl(constraint, transAlt) {
```

Comment:
It will be nice to have this function refactoreed in a Utils file somewhere, I've seen this code multiple times.

Reply:
agree, this could be standardized.